src/engine/channelmixer.h

```diff
@@ -5,21 +5,30 @@
 
 #include "util/types.h"
 #include "engine/enginemaster.h"
+#include "effects/engineeffectsmanager.h"
 
 class ChannelMixer {
   public:
-    static void mixChannels(
+    // This does not modify the input channel buffers. All manipulation of the input
+    // channel buffers is done after copying to a temporary buffer, then they are mixed
+    // to make the output buffer.
+    static void applyEffectsAndMixChannels(
```

Comment:
These functions may be able to benefit from more concise names... any suggestions?

Reply:
the name works for me.